src/modules/action-button/action-button.component.scss

```diff
@@ -0,0 +1,14 @@
+/deep/
+.stache-action-buttons {
+    display: inline-flex;
+    flex-wrap: wrap;
+
+  sky-action-button {
+    display: inline-flex;
+  }
+  
+  .sky-action-button {
+    width: 236px;
```

Comment:
Having the cards different widths made them look pretty terrible and non-uniform,  I can experiment a bit more.